daras_ai_v2/openapi_tricks.py

```
@@ -0,0 +1,69 @@
+
+
+def patch_custom_schema_fastapi(app: FastAPI):
+    if getattr(app, "_is_openapi_patched", False):
+        return
+
+    app._openapi_old = app.openapi
+
+    def custom_openapi():
+        schema = app._openapi_old()
+        loop = asyncio.get_running_loop()
+        loop.create_task(patch_openapi_full(schema))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)